packages/polymath-ui/src/redux/account/actions.js

```diff
@@ -64,6 +65,59 @@ export type AccountInnerData = {|
   email: string,
 |};
 
+const signTypedData = ({ web3, normal, typed, address }) => {
```

Comment:
Where are you using this function?

Reply:
Left it for future reference when we migrate this code. Adding a comment though!